apps/web/src/custom-pages/weekly-newsletter-page.tsx

```diff
@@ -28,12 +28,27 @@ function NewsItemContent({ content }: { content?: EditorState }) {
   );
 }
 
-function NewsSection({ newsItem }: { newsItem: NewsItem }) {
+async function NewsSection({ newsItem }: { newsItem: NewsItem }) {
+  const t = await getScopedI18n("weeklyNewsletter");
   const content = newsItem.content as unknown as EditorState;
 
   return (
     <article>
       <h3 id={stringToId(newsItem.title)}>{newsItem.title}</h3>
+      {newsItem.linkToSignUp ? (
+        <a
+          // Do not create relative links
+          href={
+            newsItem.linkToSignUp.startsWith("http")
+              ? newsItem.linkToSignUp
+              : `//${newsItem.linkToSignUp}`
+          }
+          target="_blank"
+          rel="noopener"
+        >
+          {t("link-to-sign-up")}
```

Comment:
I think we should add the event / item name here for screen reader support, so the links aren't all the same.

We can hide it from view with `sr-only` for example, but it might also make sense for sighted users

Reply:
Added the item title inside a sr-only span. Hid it from sighted users since the item title is already shown above the link, so IMO it does not have to be repeated

Reply:
Makes sense